client/gutenberg/extensions/shared/frontend-management.js

```diff
@@ -17,18 +17,19 @@ export class FrontendManagement {
 		const { selector } = options;
 		const blockClass = [ '.wp-block', name.replace( '/', '-' ) ].join( '-' );
 		rootNode.querySelectorAll( blockClass ).forEach( node => {
```

Comment:
Thanks for confirming, good for another PR since it's already in. 👍

Reply:
Ah, wasn't aware. Looking through options in https://css-tricks.com/a-bunch-of-options-for-looping-over-queryselectorall-nodelists/, it seems like the best solution looks like this:

```
const blockNodes = Array.prototype.slice.apply(
	rootNode.querySelectorAll( blockClass )
);
blockNodes.forEach( node => {
```

How does this sound?

Reply:
Done: https://github.com/Automattic/wp-calypso/pull/28795/commits/740dcb9a22bff5b6a7ed017298fdc444e94f39a0. Going to leave this unmerged for a little bit in case @simison disagrees with the approach.